city_scrapers/spiders/il_port_district.py

```diff
@@ -0,0 +1,186 @@
+import datetime
+import re
+
+import scrapy
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlPortDistrictSpider(CityScrapersSpider):
+    name = "il_port_district"
+    agency = "Illinois International Port District"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.iipd.com"]
+    location = {
+        "name": "Illinois International Port District ",
+        "address": "3600 E. 95th St. Chicago, IL 60617",
+    }
+
+    schedules_url = "https://www.iipd.com/calendar/schedules"
+
+    def start_requests(self):
+        start_urls = [
+            "https://www.iipd.com/calendar/agendas",
```

Comment:
Now that it's just two URLs it might be simpler to just do this:

```python
yield scrapy.Request(url="https://www.iipd.com/calendar/agendas", callback=self.parse_agendas)
yield scrapy.Request(url="https://www.iipd.com/about/board-meeting-minutes", callback=self.parse_minutes)
```

Reply:
Yeah, it looks much simpler and cleaner now.